app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -55,15 +55,24 @@ def download_url
       Hyrax::Engine.routes.url_helpers.download_url(representative_presenter, host: request.host)
     end
 
-    # @return [Boolean] render the UniversalViewer
-    def universal_viewer?
+    # @return [Boolean] render a IIIF viewer
+    def iiif_viewer?
       representative_id.present? &&
         representative_presenter.present? &&
         representative_presenter.image? &&
         Hyrax.config.iiif_image_server? &&
         members_include_viewable_image?
     end
 
+    alias universal_viewer? iiif_viewer?
+    deprecation_deprecate universal_viewer?: "use iiif_viewer? instead"
+
+    # Override this method to declare a different iiif viewer for your work type
```

Comment:
Could we just document that this method returns the viewer type as a symbol? I fear that saying "override" encourages folks to reopen this class, while inheriting it or implementing its interface would be better.

Reply:
WorkShowPresenter is inherited by the work presenter generated as part of a work type.  I was expecting overrides to happen at that level in the downstream app's work presenter.  What do you think would be clearer language to suggest that pattern?  Do you think the `@example` you suggest would serve this purpose?

Reply:
I know lots of folks reach straight for monkey-patching, and I'd like to move docs to language that doesn't invite that interpretation.